x/ecocredit/server/core/features/msg_create_project.feature

```diff
@@ -0,0 +1,31 @@
+Feature: CreateProject
+
+  Background:
+    Given alice has created a credit class
+
+  Rule: a custom project id must be unique
+
+    Scenario: a project id is unique
+      When alice creates a project with id "P1"
+      Then the project exists with id "P1"
+
+    Scenario: a project id is not unique
+      Given alice has created a project with id "P1"
```

Comment:
we should add a case for `001` being user selected as well to make sure the system generates `002`

Reply:
I think we should actually error if the user provides a custom project id that is not unique and prevent the user from providing a custom project idea without an alphabetical character, therefore reserving the use of project ids as number to auto-generated project ids. What do you think? @aaronc

Reply:
This would ensure the auto-generated project ids are aligned with the sequence number. We are currently not updating the sequence number if a project is created with a custom project id.

If we update the next sequence number each time a project is created (whether the project is created with an auto-generated id or custom id), we would end up with `001`, `VERRA1`, `003`.

If we allow custom numeric project ids and someone claims `003` before the next sequence number is `3`, then we would need to increment the sequence number and try creating the project again.

The sequence number is only being used for auto-generating project ids, not the total number of projects, so I think it would make more sense to require an alphabetical character when providing custom ids and to reserve numeric project ids to auto-generated projects ids.

Reply:
I now think we should remove support for custom project ids. See https://github.com/regen-network/regen-ledger/issues/1032#issuecomment-1107800763.